src/DotNet.Status.Web.Tests/DotNet.Status.Web.Tests.csproj

```diff
@@ -10,6 +10,12 @@
     <None Remove="Files\triage-items-with-diff-cat.expected.json" />
     <None Remove="Files\triage-items-with-invalid.body.txt" />
     <None Remove="Files\triage-items-with-invalid.expected.json" />
+    <None Remove="Files\triage-items-with-multi-builds-intersects.body.txt" />
```

Comment:
I think there is something inherently wrong with having a folder named `Files` 😁 something like a Hungarian notation for file system